.editorconfig

```diff
@@ -12,7 +12,6 @@ indent_style = space
 [*.{cs,csx,vb,vbx}]
 indent_size = 4
 insert_final_newline = true
-charset = utf-8-bom
```

Comment:
The opinion seems to be that BOM is redundant and doesn't help much, so removed.